scripts/abci.js

```diff
@@ -9,6 +9,16 @@ const errorHandler = require('../lib/errorHandler');
 (async function main() {
   const container = await createDIContainer(process.env);
 
+  const logger = container.resolve('logger');
+
+  logger.info('checking Core has finished syncing...');
```

Comment:
It could look strange in case if Core is synced already. Let's just remove it.

Reply:
@shumkov I believe it's handy to see it in the logs to understand where are we now. Maybe it is better to leave it?